clients/hadoopfs/src/test/java/io/lakefs/FSConfigurationTest.java

```diff
@@ -0,0 +1,47 @@
+package io.lakefs;
+
+import junit.framework.TestCase;
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Assert;
+
+public class FSConfigurationTest extends TestCase {
+
+    public void testGet() {
```

Comment:
Wooah.... `extends TestCase`?!  No `@Test`!?  Looks like [you're writing JUnit3 tests](https://stackoverflow.com/questions/2635839/junit-confusion-use-extends-testcase-or-test).  Apart from falling afoul of my innate nitpickiness, this will (probably?) prevent any tests in this class from using `@Stuff` annotations.

Can you add `@Test` and remove the inheritance?

Reply:
Sure 🥺 I gave the IDE create this one - big mistake